utils/progressbar/progressbar.go

```diff
@@ -265,7 +264,7 @@ func shouldInitProgressBar() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return terminalWidth >= minTerminalWidth, nil
+	return true, nil
```

Comment:
Following this change, the documentation above this method should be updated too:
```go
// Init progress bar if all required conditions are met:
// CI == false (or unset), and Stderr is a terminal.
```